chain-cli/chaincode-template/e2e/launchpad-api.spec.ts

```diff
@@ -1146,7 +1125,7 @@ describe("LaunchpadContract", () => {
       );
     });
 
-    test("SellWithExactTokens || It should revert if Expected Gala tokens from this operation exceeds the actual amount that will be provided.", async () => {
+    test("SellWithExactTokens || It should rever if Expected Gala tokens from this operation exceeds the actual amount that will be provided.", async () => {
```

Comment:
```suggestion
    test("SellWithExactTokens || It should revert if Expected Gala tokens from this operation exceeds the actual amount that will be provided.", async () => {
```

Unintentional typo introduced?